### PR TITLE
fix: copyright clarification for modular production KIT

### PR DESCRIPTION
**Pull request by dominikoeh**
## Description

Clarification of copyright notice in the Modular Production KIT. The current release states 

> `Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)`

where it should have been 

> `Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)`

@wehrstedt-jan 

@danielmiehle @maximilianong 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by FaGru3n** [comment]
hi @dominikoeh thanks for your contribution, did also checked the Dependencies and licence information?

because it is not check marked? and also a little info you can request reviewers by adding them in the reviewer section 😉

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — COMMENTED** [review]
Please no changes on the 23.12 - this one is released. You can change the file in the "next".

**Review by maximilianong — APPROVED** [review]
Talked to @danielmiehle - change is approved. 
No content change.

**Review by wehrstedt-jan — APPROVED** [review]

File: docs-kits/kits/Modular Production Kit/page_adoption-view.md [hunk 1]
```diff
@@ -84,5 +84,5 @@ The standards for release 3.2 will be published soon. Our relevant standards can
 This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
 
 - SPDX-License-Identifier: CC-BY-4.0
-- SPDX-FileCopyrightText: 2023,Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)
+- SPDX-FileCopyrightText: 2023;Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
 - SPDX-FileCopyrightText: 2023,Siemens AG
```
Comment (FaGru3n) [inline docs-kits/kits/Modular Production Kit/page_adoption-view.md:88]:
```suggestion
- SPDX-FileCopyrightText: 2023 Siemens AG
```

File: docs-kits/kits/Modular Production Kit/page_adoption-view.md [hunk 2]
```diff
@@ -84,5 +84,5 @@ The standards for release 3.2 will be published soon. Our relevant standards can
 This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
 
 - SPDX-License-Identifier: CC-BY-4.0
-- SPDX-FileCopyrightText: 2023,Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)
+- SPDX-FileCopyrightText: 2023;Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
```
Comment (FaGru3n) [inline docs-kits/kits/Modular Production Kit/page_adoption-view.md:87]:
```suggestion
- SPDX-FileCopyrightText: 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
```

following the tips and tricks on one of your  [TRG 7-7](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07#documentation)

File: docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/page_adoption-view.md [hunk 3]
```diff
@@ -84,5 +84,5 @@ The standards for release 3.2 will be published soon. Our relevant standards can
 This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
 
 - SPDX-License-Identifier: CC-BY-4.0
-- SPDX-FileCopyrightText: 2023,Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)
+- SPDX-FileCopyrightText: 2023;Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
```
Comment (FaGru3n) [inline docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/page_adoption-view.md:87]:
```suggestion
- SPDX-FileCopyrightText: 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
```

File: docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/page_adoption-view.md [hunk 4]
```diff
@@ -84,5 +84,5 @@ The standards for release 3.2 will be published soon. Our relevant standards can
...
 - SPDX-FileCopyrightText: 2023,Siemens AG
```
Comment (FaGru3n) [inline docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/page_adoption-view.md:88]:
```suggestion
- SPDX-FileCopyrightText: 2023 Siemens AG
```

**Review by FaGru3n — CHANGES_REQUESTED** [review]
Hi, looke quite ok but some symbols seems a bit confusing why to change that in that way

FYI @AngelikaWittek,

could you maybe give us a hint which is the proper format in this way?

**Review by FaGru3n — APPROVED** [review]
LGTM

**Review by maximilianong — APPROVED** [review]

**Pull request merged**